server/index.js

```diff
@@ -37,9 +37,10 @@ app.get('/products/:productId', async (req, res) => {
 // get all available stock using raw SQL query with inner joins
 app.get('/stock', async (req, res) => {
 
-  const stocks = await sequelize.query("SELECT stocks.id, products.name, \
-  stores.location, stocks.color, stocks.size, stocks.qty, products.id as \
-  productId FROM stocks INNER JOIN stores ON stores.id = stocks.storeId \
+  const stocks = await sequelize.query("\
```

Comment:
You should be able to do these queries using the model interface with eager loading. Is this somethign you tried already?

https://sequelize.org/master/manual/eager-loading.html

It allows you to make use of the ORM instead of write SQL here in your javascript.